### PR TITLE
mount user to jenkins

### PR DESCRIPTION
**Pull request by krzyzacy**
let's try again

### CODE REVIEW

**Comment by Random-Liu** [comment]
@krzyzacy You need to create the user `jenkins` in Dockerfile. Or else you will get the error:
```
docker: Error response from daemon: linux spec user: unable to find user jenkins: no matching entries in passwd file.
```

**Comment by Random-Liu** [comment]
NVM. If the host has `jenkins` user, it also works.

**Comment by krzyzacy** [comment]
Jeff suggested remove `--privileged=true` might make it run as jenkins, also mapped credential to user jenkins, worth trying.

**Comment by krzyzacy** [comment]
merge to try a last run before the darkness falls

File: jenkins/node-dockerized.sh [hunk 1]
```diff
@@ -40,13 +40,12 @@ mkdir -p "${HOST_ARTIFACTS_DIR}"
...
 
 docker run --rm=true \
-  --privileged=true \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -v "${REPO_DIR}":/go/src/k8s.io/kubernetes \
   -v "${WORKSPACE}/_artifacts":/workspace/artifacts \
   -v /etc/localtime:/etc/localtime:ro \
-  ${JENKINS_GCE_SSH_PRIVATE_KEY_FILE:+-v "${JENKINS_GCE_SSH_PRIVATE_KEY_FILE}:/root/.ssh/google_compute_engine:ro"} \
-  ${JENKINS_GCE_SSH_PUBLIC_KEY_FILE:+-v "${JENKINS_GCE_SSH_PUBLIC_KEY_FILE}:/root/.ssh/google_compute_engine.pub:ro"} \
+  ${JENKINS_GCE_SSH_PRIVATE_KEY_FILE:+-v "${JENKINS_GCE_SSH_PRIVATE_KEY_FILE}:/jenkins/.ssh/google_compute_engine:ro"} \
```
Comment (ixdy) [inline jenkins/node-dockerized.sh]: this is probably not the right homedir. we'll see what happens?

Comment (Random-Liu) [inline jenkins/node-dockerized.sh]: Even `non-privileged`, we are still login as "root".

**Pull request merged**